amqp-transport-binding.md

```diff
@@ -236,12 +236,12 @@ content-type: application/cloudevents+json; charset=UTF-8
 
 #### 3.2.2. Event Data Encoding
 
-The chosen [event format](#14-event-formats) defines how all attributes,
-including the `data` attribute, are represented.
+The chosen [event format](#14-event-formats) defines how all attributes and the
+data payload are represented.
 
-The event metadata and data is then rendered in accordance with the event format
-specification and the resulting data becomes the AMQP application [data][data]
-section.
+The event metadata and payload  is then rendered in accordance with the event
```

Comment:
`s/payload  /payload /`

I'm also not sure why you've dropped `data` here, shouldn't it be `data payload`?